### ISSUE TITLE
[Feature Request] Add brighter color option for dark dynamic theme

### ISSUE THREAD

**Issue by Lowell-Syncrety** [darkreader/darkreader#3219]
I'm guessing the goal of DarkReader is more to help with reading in a dark environment rather than "pretty-izing" sites into a cool-looking dark theme and that's OK. I'm using it as the latter option (also because it's easier on the eyes at any time) and overall it's great for that! In some cases though, I think it's more when the original site colors/highlights are lighter or more pastel in nature, the colors turn out to be dark, drab versions. 

I wonder if there's a way to add a "bright color" mode to the dynamic theme. I only see this in a few places, like in this scorecard from BrightGauge. Below at left is the regular site, then with DarkReader turned on:
![BG](https://user-images.githubusercontent.com/40280476/88350575-5eac3d00-cd08-11ea-84f3-ef6722c43836.jpg)

**Comment by Gusted** [comment]
Let me clarify some things.
dark reader purpose is to make site a dark mode version of it. And dynamic is built to analyze sheets and colors and convert them to make the site a good looking dark mode. Second the picture your showing has been processed by dark reader to make them look more natural in the dark mode version because imagine the light colors into a dark environment, that looks bad and can say from experience. I recommend to give it a play with the current options or revert the colors manually trough a extension like stylus.

**Comment by Lowell-Syncrety** [comment]
Yeah, I realize that's the goal of DarkReader, just thought I'd add a feature request that would brighten up some of the colors. For example here in Github the green in the "Comment" and "New issue" buttons isn't dull and dark; it's relatively similar to the original green color. Of course in my example you don't want a light pastel green in a dark screen (who'd even want the light pastel green to begin with, but I digress).

I just did a bit of experimenting and it looks like DarkReader is just literally darkening those pastel colors by a significant percentage. If it was a lower percentage they wouldn't be as dark. Is there a way to customize how much dynamic mode adjusts color brightness for a specific site?

I'll have a look at stylus; haven't heard of that, thanks.

**Comment by Gusted** [comment]
Oh if you want something that doesn't analyze colors you should look at filter mode that doesn't care about colors but just white and black colors. You can choose just for x site and than play with the options.

**Comment by Lowell-Syncrety** [comment]
Oh, no, it's great that DarkReader analyzes colors. Just curious if there's a way to adjust the amount of darkness that it's applying to colors on a site-by-site basis. So for this particular site I'd adjust it to darken maybe by 50% instead of 80%.

**Comment by Gusted** [comment]
Kind of check version 5 preview
https://darkreader.org/blog/v4-9/

**Comment by Lowell-Syncrety** [comment]
I've been using the preview; great update overall. Relevant to this request however, the update only allows for changing colors of the background, text, scrollbar, or selection. My ask is for one more option, to provide a % value for overall dynamic color shading. Currently it appears it's darkening light colors by about 80%, for example. For my preference I'd change that to about 50%. Or maybe it's a "dynamic range" slider/value that could be added.

**Comment by Gusted** [comment]
Okay,

Text = this is in our code base `color:` and `fill:` So every element that use color or fill is being modified trough the custom setting.
Background = `background-color:` So every element that use a background color is also being modified trough the custom setting.
So the majority of web page elements is being adjusted to the preffered color.
Color shading? Just modify the Background/Text option that's what's being used to modify the original color to but still intacts the `old color`. Darkening light colors, because we need to light colors in dark background envirement it's not readable, [this happends than](https://i.imgur.com/1wgVUxV.png) I can't see what's on the button :)

**Comment by Lowell-Syncrety** [comment]
Ah, I see. So colored elements of a web page that we can't directly change, change dynamically based on the color/darkness of the background. So in order to get brighter colored elements I have to brighten the background. There's just no slider/option to adjust the amount of how much other elements get darkened when the background gets darkened. That is, if I set the background color to light gray then the elements brighten up. If I set the background to almost black, the colors darken a lot.

In the screenshot below, the first example at the left is the default dark theme. The next one in the middle is brightening the background to light gray, and darkening the text. The colors are what I'd like to see here, but with a dark background like the third screenshot in the right, where I just manually pasted the dark background at the top over the light gray. If this is not possible, no problem. I just though there could be an option to reduce the dynamic color adjustment when darkening the background so I'd get a similar effect to the screenshot on the right.
![BG2](https://user-images.githubusercontent.com/40280476/88830413-c4724c00-d182-11ea-99d7-63da1e09a865.jpg)
